<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBoxSetup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Setup</string>
    </property>
    <widget class="QWidget" name="layoutWidget1">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>20</y>
       <width>179</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelPort">
        <property name="text">
         <string>Port #:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxPort"/>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxEnabled">
        <property name="text">
         <string>Enabled</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>371</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labelFrequency">
        <property name="text">
         <string>Frequency:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditFrequency"/>
      </item>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QPushButton" name="pushButtonSetFrequency">
         <property name="text">
          <string>Set</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonGetFrequency">
         <property name="text">
          <string>Get</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButtonOpenDevice">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Device</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>371</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="labelMaxReplyCount">
        <property name="text">
         <string>Max Reply Count:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditMaxReplyCount"/>
      </item>
      <item>
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QPushButton" name="pushButtonSetMaxReplyCount">
         <property name="text">
          <string>Set</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonGetMaxReplyCount">
         <property name="text">
          <string>Get</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxTransfer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>391</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Transfer</string>
    </property>
    <widget class="QTextEdit" name="textEditWrite">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>181</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditRead">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>181</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget2">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>20</y>
       <width>342</width>
       <height>77</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelSlaveAddress">
        <property name="text">
         <string>Slave Address:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxSlaveAddress">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonScanAddresses">
        <property name="text">
         <string>Scan</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelMemoryAddress">
        <property name="text">
         <string>Memory Address:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditMemoryAddress"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelMemAddrLength">
        <property name="text">
         <string>Memory Address Length:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxMemAddrLength">
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>130</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelBufferSize">
        <property name="text">
         <string>Buffer size:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditBufferSize"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>158</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonWrite">
        <property name="text">
         <string>Write</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButtonRead">
        <property name="text">
         <string>Read</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
